app/representers/api/v1/unclaimed_user_representer.rb

```diff
@@ -0,0 +1,13 @@
+module Api::V1
+  class UnclaimedUserRepresenter < Roar::Decorator
```

Comment:
If you have to reuse the existing ApplicationUserRepresenter, you will need to add `:if` conditions on most fields to prevent them from showing for certain states.

I don't think we want the ApplicationUserRepresenter, though, since there is no ApplicationUser until you authorize the app to read your info. If anything, this could be the UserRepresenter (with extra `:if`'s on most fields.